### PR TITLE
BCM2712d0 support

### LINKED ISSUE #795: [BUG] Embedded: Pi 4: Imager doesn't see my USB SATA drive
**Describe the bug**
Pi 4B 8GB. Running Network Install (embedded Imager) to do a reinstall on my USB-connected SATA drive. Bridge is ASM1153 to a Kingston A400 series 2.5" SATA drive.

**To Reproduce**
1. Boot Pi 4 with no boot media present and Ethernet connected.
2. Hold shift to enter Network Install.
3. Plug in USB SATA drive.
4. Click 'Choose Storage' button in Imager.

or

1. Boot Pi 4 with USB SATA drive present but no Pi boot files present.
2. Hold shift to enter Network Install.
3. Click 'Choose Storage' button in Imager.

**Expected behaviour**
Drive shows up in Imager's 'Storage' dialog. Can image drive OK.

**Actual behaviour**
Imager's 'Storage' dialog is empty. Cannot image drive using embedded Imager.

### PR DESCRIPTION
**Pull request by tdewey-rpi**
Uprate the buildroot distribution we use for the embedded imager, selecting a new kernel, firmware and mesa in the process.

Tested on BCM2712d0.

Resolves #795

### CODE REVIEW

**Comment by tdewey-rpi** [comment]
Holding review requests until I've rebuilt cleanly from this PR, and confirmed it boots on D0 and C1 variants of BCM2712.

**Comment by tdewey-rpi** [comment]
Confirmed that the updated PR builds, boots on C1 and D0.

File: embedded/imager/configs/rpi-imager_defconfig [hunk 1]
```diff
@@ -1,45 +1,3993 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Buildroot -gec35dcf Configuration
+#
+BR2_HAVE_DOT_CONFIG=y
+BR2_EXTERNAL_NAMES="IMAGER"
+BR2_EXTERNAL_IMAGER_PATH="/home/tdewey/src/rpi-imager/embedded/imager"
```
Comment (lurch) [inline embedded/imager/configs/rpi-imager_defconfig:7]: Will this still build for users that don't have a `/home/tdewey/...` directory on their computer?

Comment (tdewey-rpi) [reply embedded/imager/configs/rpi-imager_defconfig:7]: I'll prune it, but I would expect it to be overridden at build-time. The correct way to build the Embedded Imager is to use the build.sh file, which should override these BR2_EXTERNAL directives with the buildroot invocation:

`make -C "${BUILDROOT}" BR2_EXTERNAL="${PWD}/imager" rpi-imager_defconfig`

Comment (lurch) [reply embedded/imager/configs/rpi-imager_defconfig:7]: Ahhh, awesome; I didn't realise that :+1:

**Pull request merged**